packages/dd-trace/src/config.js

```diff
@@ -79,6 +80,15 @@ class Config {
       process.env.DD_RUNTIME_METRICS_ENABLED,
       false
     )
+    let DD_DBM_PROPAGATION_MODE = coalesce(
+      options.dbmPropagationMode,
+      process.env.DD_DBM_PROPAGATION_MODE,
+      'disabled'
+    )
+    if (DD_DBM_PROPAGATION_MODE === 'full') {
+      DD_DBM_PROPAGATION_MODE = 'service'
+      logger.debug('The `full` mode for DBM propagation is not yet supported. Falling back to `service`.')
```

Comment:
Actually, looking closer, this will happen before the logger is configured and thus impossible to ever see. I'm making a related change here which could perhaps be pulled out to a separate PR: https://github.com/DataDog/dd-trace-js/pull/2477/commits/3b4da808e753b3c60c172a3f480b27b19b5c320a#diff-35b625667f1fbe3cfe5d2d1290484d9643c044d90ece6dea383d9e26a1633755R63-R75

Reply:
Opened a PR for that: https://github.com/DataDog/dd-trace-js/pull/2505

Reply:
Actually, isn't it supported now anyway?

Reply:
Yeah, seems like the error is out of date.

Reply:
Good point. This whole `if` statement should be removed.

Reply:
very true removing the if block